packages/kit/src/views/Market/MarketDetailV2/components/SwapPanel/components/TokenInputSection/TokenList.tsx

```
@@ -73,16 +75,21 @@ export function TokenList({
         const networkConfig = Object.values(presetNetworksMap).find(
           (n) => n.id === token.networkId,
         );
-
+        const priceBN = new BigNumber(swapTokenDetail?.price || 0);
+        const balanceBN = new BigNumber(swapTokenDetail?.balanceParsed || 0);
+        const valueProps =
+          swapTokenDetail?.price && parseFloat(swapTokenDetail.price) > 0
```

Biome (1.9.4)

[error] Use Number.parseFloat instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseFloat instead.


(lint/style/useNumberNamespace)